format.sh

```diff
@@ -1,18 +1,10 @@
-#!/usr/bin/env nix-shell
```

Comment:
The idea is that one would never directly run this script, and instead run `nix run .#format`

https://github.com/tweag/ormolu/blob/3e7308c57ad13b315e31c6dad132c9883e2b15a5/.github/workflows/ci.yml#L46

which is defined here

https://github.com/tweag/ormolu/blob/3e7308c57ad13b315e31c6dad132c9883e2b15a5/flake.nix#L117-L118

and [`writeShellApplication`](https://noogle.dev/preview?fn=./pkgs/build-support/trivial-builders.nix/writeShellApplication) already adds a shebang.

Should I move `format.sh` in the `nix` subdir to maybe make this more clear?

Reply:
Yes, I think it would help, especially for people who are used to the old way of doing things! Can you also please update `CONTRIBUTING.md`  accordingly?